### PR TITLE
add goplus locker fee

### PR DESCRIPTION
**Pull request by goplusbot**

### CODE REVIEW

**Comment by llamabutler** [comment]
The goplus-locker.ts adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts fees goplus-locker.ts
    
    🦙 Running GOPLUS-LOCKER.TS adapter 🦙
    ---------------------------------------------------
    Start Date:	Tue, 17 Dec 2024 03:51:32 GMT
    End Date:	Wed, 18 Dec 2024 03:51:32 GMT
    ---------------------------------------------------
    
    Error: INDEXA_DB not set
        at getConnection (/home/runner/work/dimension-adapters/dimension-adapters/helpers/indexer.ts:10:24)
        at queryIndexer (/home/runner/work/dimension-adapters/dimension-adapters/helpers/indexer.ts:30:22)
        at fetch (/home/runner/work/dimension-adapters/dimension-adapters/fees/goplus-locker.ts:28:45)
        at processTicksAndRejections (node:internal/process/task_queues:95:5)
        at async getChainResult (/home/runner/work/dimension-adapters/dimension-adapters/adapters/utils/runAdapter.ts:51:18)
        at async Promise.all (index 4)
        at async runAdapter (/home/runner/work/dimension-adapters/dimension-adapters/adapters/utils/runAdapter.ts:27:20)
        at async /home/runner/work/dimension-adapters/dimension-adapters/cli/testAdapter.ts:84:21 {
      chain: 'gravity'
    }

**Comment by llamabutler** [comment]
The goplus-locker.ts adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts fees goplus-locker.ts
    
    🦙 Running GOPLUS-LOCKER.TS adapter 🦙
    ---------------------------------------------------
    Start Date:	Tue, 17 Dec 2024 03:58:42 GMT
    End Date:	Wed, 18 Dec 2024 03:58:42 GMT
    ---------------------------------------------------
    
    Error: INDEXA_DB not set
        at getConnection (/home/runner/work/dimension-adapters/dimension-adapters/helpers/indexer.ts:10:24)
        at queryIndexer (/home/runner/work/dimension-adapters/dimension-adapters/helpers/indexer.ts:30:22)
        at fetch (/home/runner/work/dimension-adapters/dimension-adapters/fees/goplus-locker.ts:28:45)
        at processTicksAndRejections (node:internal/process/task_queues:95:5)
        at async getChainResult (/home/runner/work/dimension-adapters/dimension-adapters/adapters/utils/runAdapter.ts:51:18)
        at async Promise.all (index 0)
        at async runAdapter (/home/runner/work/dimension-adapters/dimension-adapters/adapters/utils/runAdapter.ts:27:20)
        at async /home/runner/work/dimension-adapters/dimension-adapters/cli/testAdapter.ts:84:21 {
      chain: 'ethereum'
    }

File: fees/goplus-locker.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  let feeTo = "0x521faAcDFA097ad35a32387727e468F7fD032fD6";
+
+  await options.api.sumTokens({ owner: feeTo, token: ADDRESSES.null });
+  await options.fromApi.sumTokens({ owner: feeTo, token: ADDRESSES.null });
+  dailyFees.addBalances(options.api.getBalancesV2());
+  dailyFees.subtract(options.fromApi.getBalancesV2());
+
+  if( options.chain in CHAIN_CONFIG) {
+    let startBlock = CHAIN_CONFIG[ options.chain as keyof typeof CHAIN_CONFIG].start;
+    const transfer_logs = await queryIndexer(`
```
Comment (dtmkeng) [inline fees/goplus-locker.ts]: our index it support only eth chain

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by dtmkeng** [comment]
@goplusbot  goplus-locker is it sub protocol of this https://defillama.com/protocol/goplus-security#information ?

**Comment by dtmkeng** [comment]
okay will list

**Pull request merged**